src/OrleansServiceBus/ErrorCode.cs

```diff
@@ -0,0 +1,57 @@
+﻿
+using System;
+using Orleans.Runtime;
+
+namespace Orleans.ServiceBus
+{
+    /// <summary>
+    /// Orleans ServiceBus error codes
+    /// </summary>
+    internal enum OrleansServiceBusErrorCode
+    {
+        /// <summary>
+        /// Start of orlean servicebus errocodes
+        /// </summary>
+        ServiceBus = 1<<16,
+
+        FailedPartitionRead = ServiceBus,
```

Comment:
Not sure if it's an actual convention or not, but I noticed that everywhere where we have error code constants, we skip the feature base code (meaning that this should have been `FailedPartitionRead = ServiceBus + 1`)